docs/user-guide/faq.md

```diff
@@ -0,0 +1,67 @@
+# FAQ
+
+## How do I lint within my text editor?
+
+The stylelint community maintains a [handful of plugins](/docs/user-guide/complementary-tools.md#editor-plugins) for popular editors. Refer to their individual READMEs to get started.
+
+## How do I lint from the command line?
+
+First, install stylelint using the `-g` option:
+
+```shell
+npm install -g stylelint
+```
+
+Then refer to the [CLI usage examples](/docs/user-guide/cli.md) within the doc.
+
+The CLI can also be used from within [npm run scripts](http://blog.keithcirkel.co.uk/how-to-use-npm-as-a-build-tool/) if you'd rather use them over a task runner like `gulp`.
```

Comment:
How about

`The CLI can also be used from within [npm run scripts](http://blog.keithcirkel.co.uk/how-to-use-npm-as-a-build-tool/) to use a non-global installation of stylelint.`